runtime/manta/src/lib.rs

```diff
@@ -844,7 +844,7 @@ pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, RuntimeCall, Si
 
 /// Types for runtime upgrading.
 /// Each type should implement trait `OnRuntimeUpgrade`.
-pub type OnRuntimeUpgradeHooks = migrations::assets_genesis::AssetsGenesis<Runtime>;
+pub type OnRuntimeUpgradeHooks = ();
```

Comment:
Pretty sure this migration already happened

Reply:
yes